cypress/integration/fileMenu.cy.js

```diff
@@ -0,0 +1,153 @@
+import { typeInput } from '../helpers/common.js'
+import { selectEventProgram } from '../helpers/dimensions.js'
+import { clickMenubarUpdateButton } from '../helpers/menubar.js'
+import { EXTENDED_TIMEOUT } from '../support/util.js'
+
+const ITEM_NEW = 'file-menu-new'
+const ITEM_OPEN = 'file-menu-open'
+const ITEM_SAVE = 'file-menu-save'
+const ITEM_SAVEAS = 'file-menu-saveas'
+const ITEM_RENAME = 'file-menu-rename'
+const ITEM_TRANSLATE = 'file-menu-translate'
+const ITEM_SHARING = 'file-menu-sharing'
+const ITEM_GETLINK = 'file-menu-getlink'
+const ITEM_DELETE = 'file-menu-delete'
+
+const defaultItemsMap = {
+    [ITEM_NEW]: false,
+    [ITEM_OPEN]: false,
+    [ITEM_SAVE]: false,
+    [ITEM_SAVEAS]: false,
+    [ITEM_RENAME]: false,
+    [ITEM_TRANSLATE]: false,
+    [ITEM_SHARING]: false,
+    [ITEM_GETLINK]: false,
+    [ITEM_DELETE]: false,
+}
+
+const assertItems = (enabledItemsMap) => {
+    const itemsMap = Object.assign({}, defaultItemsMap, enabledItemsMap)
+
+    Object.keys(itemsMap).forEach((itemName) => {
+        itemsMap[itemName]
```

Comment:
if you're accessing both the key (through the prop] and the value (through `itemsMap[x]`) you might as well use `Object.entries` that does this for you rather than `Object.keys` and looping through the array again...

Reply:
i.e. 
```
Object.entries(itemsMap).forEach(([key, value]) => {
        value 
        ? cy.getBySel(key).should...
```

Reply:
Object.keys is faster, so keeping it.

Reply:
Updated